src/aics-image-viewer/components/Toolbar/index.tsx

```diff
@@ -0,0 +1,182 @@
+import React from "react";
+import { Button, Dropdown, Icon, Menu, Radio, Select, Tooltip } from "antd";
+import "./styles.css";
+
+import ViewModeRadioButtons from "../ViewModeRadioButtons";
+
+import viewMode from "../../shared/enums/viewMode";
+import {
+  FULL_FIELD_IMAGE,
+  MAX_PROJECT,
+  PATH_TRACE,
+  SEGMENTED_CELL,
+  THREE_D_MODE,
+  VOLUMETRIC_RENDER,
+} from "../../shared/constants";
+
+interface DownloadButtonProps {
+  cellDownloadHref: string;
+  fovDownloadHref: string;
+  hasFov: boolean;
+}
+
+interface ToolbarProps {
+  imageName: string;
+  imageType: string;
+  renderSetting: string;
+  cellDownloadHref: string;
+  fovDownloadHref: string;
+  mode: symbol;
+  hasCellId: boolean;
+  hasParentImage: boolean;
+  autorotate: boolean;
+  pathTraceOn: boolean;
+  canPathTrace: boolean;
+  showAxes: boolean;
+  showBoundingBox: boolean;
+
+  onViewModeChange(mode: symbol): void;
+  onResetCamera(): void;
+  onAutorotateChange(): void;
+  onSwitchFovCell(value: string): void;
+  onChangeRenderingAlgorithm(newAlgorithm: string): void;
+  changeAxisShowing(showing: boolean): void;
+  changeBoundingBoxShowing(showing: boolean): void;
+
+  renderConfig: {
+    autoRotateButton: boolean;
+    viewModeRadioButtons: boolean;
+    fovCellSwitchControls: boolean;
+    resetCameraButton: boolean;
+    showAxesButton: boolean;
+    showBoundingBoxButton: boolean;
+  };
+}
+
+function DownloadButton({ fovDownloadHref, cellDownloadHref, hasFov }: DownloadButtonProps) {
```

Comment:
I suggest moving DownloadButton into its own file within this directory. I know it's very short but it's sort of right on the edge of worth moving out.  Also it's good to aim for one component per file